CHANGELOG.md

```diff
@@ -1,13 +1,14 @@
 # Changelog
 
-## Next version
+## 23.11.0
```

Comment:
Could potentially add in a new section: `Upcoming Breaking Changes` to mention the upcoming removal of the deprecated option?

Reply:
added 👍🏼